dotcom-rendering/src/components/BylineLink.tsx

```diff
@@ -141,6 +146,41 @@ export const BylineLink = ({
 		);
 	});
 
+	if (
+		(design === ArticleDesign.Video || design === ArticleDesign.Audio) &&
+		source
+	) {
+		renderedTokens.push(
+			<>
+				{renderedTokens.length > 0 && ', '}
+				<Source key="source" source={source} />
+			</>,
+		);
```

Comment:
If you’re the kind of person that wants to avoid mutation, you could do:

```suggestion
		return renderedTokens.concat(
			<>
				{renderedTokens.length > 0 && ', '}
				<Source key="source" source={source} />
			</>,
		);
```